site/content/guides/monitor-many-urls.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Monitor 500 pages in 2 minutes with Checkly
+displayTitle: Monitor 500 pages in 2 minutes with Checkly
```

Comment:
I found the headline slightly misleading when reading it for the first time. "Creating monitors" would be more tangible if we want to be explicit.